bgpd/bgp_mac.c

```diff
@@ -311,6 +311,23 @@ void bgp_mac_del_mac_entry(struct interface *ifp)
 	bgp_mac_remove_ifp_internal(bsm, ifp->name);
 }
 
+bool bgp_mac_exist(struct ethaddr *mac)
```

Comment:
As there is already an API bgp_mac_entry_exist for similar functionality, either adding a comment explaining what this new function does, to give more details on how this differs from existing API, or considering to rename this API to make it more explicit will be useful. You can take the call.

Reply:
I have pushed the change explaining the usage of the APIs.